Backup/minimizing-restore-costs.md

```diff
@@ -0,0 +1,19 @@
+{{{
+  "title": "Minimizing Restore Costs",
+  "date": "02-17-2016",
+  "author": "Justin Withington",
+  "attachments": [],
+  "related-products" : [],
+  "contentIsHTML": false,
+  "sticky": false
+}}}
+
+**Minimizing Restore Costs**
+
+We understand that optimization of your budget is important and are constantly working towards offering an effective service at a reduced price. In conjunction with this goal, roadmap items exist for lowering overall restore costs and providing partial restores to help minimize the the return of unnecessary data during a restore. In addition, there are a few tips and tricks that are able to be utilized to assist with minimizing restore costs.
```

Comment:
should we really talk about roadmap items in a public kb?  we should talk about items in place.  particularly saying we will lower price....not recommended from the view of our Sr. CSA team.  just focus on tips and tricks not some future thing that may or may not happen.